features/netsocket/emac-drivers/TARGET_STM/stm32xx_emac.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2017 ARM Limited. All rights reserved.
```

Comment:
not a license we use (apache, and its common in most of the mbed files) ?

Reply:
Corrected.